src/ducks/billsMatching/Linker/billsToOperation/operationsFilters.spec.js

```diff
@@ -140,14 +140,9 @@ describe('operations filters', () => {
     test('not health bill', () => {
       const fByCategory = filterByCategory({ vendor: 'SFR' })
       expect(fByCategory({ manualCategoryId: HEALTH_EXPENSE_CAT })).toBeFalsy()
-      expect(
-        fByCategory({ automaticCategoryId: HEALTH_EXPENSE_CAT })
-      ).toBeFalsy()
       expect(fByCategory({ manualCategoryId: '0' })).toBeTruthy()
-      expect(fByCategory({ automaticCategoryId: '0' })).toBeTruthy()
       expect(fByCategory({})).toBeTruthy()
       expect(fByCategory({ manualCategoryId: '400611' })).toBeTruthy()
-      expect(fByCategory({ automaticCategoryId: '400611' })).toBeTruthy()
```

Comment:
For my information, why did you remove those lines ?

Reply:
Because the case in which we only have `automaticCategoryId` means that our categorization models have not been applied, and in that case the `getCategoryId` helper returns `null`. That's how we represent the "categorization pending" state when a transaction have been improted but not yet categorized by our categorization service.

In tests, I think it's better to use only `manualCategoryId` because when it is present, it is always used. So we don't have to think about all the specific cases implemented by `getCategoryId` in every test that uses categories. By always using `manualCategoryId`, we are certain that this category will be used.